### PR TITLE
Change |ConfigDiffBuildable| to |InputConfigDiffBuildable| with more parameters.

### PR DESCRIPTION
**Pull request by dmikurube**
@muga Updating `ConfigDiffBuildable` to take more parameters.

### CODE REVIEW

File: src/main/java/org/embulk/base/restclient/InputConfigDiffBuildable.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package org.embulk.base.restclient;
+
+import org.embulk.config.ConfigDiff;
+import org.embulk.spi.InputPlugin;
+import org.embulk.spi.Schema;
+
+public interface InputConfigDiffBuildable<T extends RestClientInputTaskBase>
+{
+    ConfigDiff buildInputConfigDiff(T task, Schema schema, int taskCount, InputPlugin.Control control);
```
Comment (muga) [inline src/main/java/org/embulk/base/restclient/InputConfigDiffBuildable.java]: I think that `control` is not necessary. We need `List<TaskReport>` object that `control` returns instead.

**Review by muga — COMMENTED** [review]
I put one minor comment. Please check it. The rest looks good to me 👍

File: src/main/java/org/embulk/base/restclient/InputConfigDiffBuildable.java (hunk 1 above)
Comment (dmikurube) [reply src/main/java/org/embulk/base/restclient/InputConfigDiffBuildable.java]: Ah, right. Fixed!

**Review by muga — COMMENTED** [review]
Thank you for fixing. LGTM 👍

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**